doc/INSTALL.md

```diff
@@ -53,6 +53,8 @@ The `etc/apt/source.list` entry must look like following: `deb https://debs.libe
 
 E.g. `deb https://debs.libelektra.org/focal focal-unstable main`
```

Comment:
```suggestion
   E.g. `deb https://debs.libelektra.org/focal focal-unstable main`
```

IMO it would be better to also indent the section below point `2` to make it obvious that it belongs to  `2` and enhance the readability.